### PR TITLE
export github access token

### PR DESCRIPTION
**Pull request by olebedev**
### Motivation

As per discussion, exporting a github access token as a separate env var.
/cc @uri-canva

### CODE REVIEW

**Review by arromer — APPROVED** [review]

File: hooks/pre-checkout [hunk 1]
```diff
@@ -62,6 +62,7 @@ export_buildkite_repo() {
   local token
   token=$(jq -r '.body' <"${out_file}" | jq -r '.token')
 
+  export GITHUB_ACCESS_TOKEN="${token}"
```
Comment (uri-canva) [inline hooks/pre-checkout:65]: Can this have a more specific name? I think a lot of scripts will have their own `GITHUB_ACCESS_TOKEN` if they use the github API, but this token should only be used to fetch repos.

Comment (uri-canva) [reply hooks/pre-checkout:65]: Yes.

Comment (olebedev) [inline hooks/pre-checkout:65]:
```suggestion
  export GITHUB_FETCH_APP_ACCESS_TOKEN="${token}"
```

Comment (olebedev) [reply hooks/pre-checkout]: Done in 0ca2f4ee3c26078b328c4a645991e2496aabdbda

**Review by arromer — APPROVED** [review]

**Comment by olebedev** [comment]
I can't see a merge button for some reason, maybe because I am sending PR from my fork or something like that. @arromer, can you please merge it?

**Pull request merged**